src/Analysim.Web/ClientApp/src/app/projects/project-overview/project-overview-view/project-content/modal-datasets/modal-datasets.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Notebook } from 'src/app/interfaces/notebook';
+import { Project } from 'src/app/interfaces/project';
+import { ProjectService } from 'src/app/services/project.service';
+
+@Component({
+  selector: 'app-modal-datasets',
+  templateUrl: './modal-datasets.component.html',
+  styleUrls: ['./modal-datasets.component.scss']
+})
+export class ModalDatasetsComponent implements OnInit {
+
+  isLoading: Boolean;
+  fileTree: any;
+
+  @Output() closeModal: EventEmitter<any> = new EventEmitter();
+
+  @Input() notebook: Notebook;
+  @Input() project: Project;
+
+  blobFiles = [
+    {
+      directory: "test/",
```

Comment:
are these test data? should they stay here?

Reply:
yeah , they were test data , I have removed them , thanks for noticing .